### ISSUE TITLE
Add "reason" field to training data outputs

### ISSUE THREAD

**Issue by dcorney** [FullFact/health-misinfo-shared#1]
### Overview

Currently, the fine-tuning task is to take a chunk of text and return a list of checkworthy health claims, i.e. a simple list of strings. The training set therefore is a list of chunks (input) and checkworthy claims (target output).

However, we have expert knowledge about what makes some claims checkworthy (e.g. they are concrete claims, they refer to studies or science, they contradict clear scientific advice etc.) which we should include in our training. We can do this by making the target output richer and fine-tuning on that. Claims containing "hedge words", like "can help", "may be effective", "possibly useful" etc. are generally not worth checking.

### Requirements

1. vertex.py` generate_training_set()` should be updated so that the CSV file it produces is easier to label. E.g.  produce a separate file with just the claim field. When this is labelled in an external spreadsheet, we'll need to import it again. So...
2. update the fine_tuning.py `make_training_set()` so that it reads in the annotated CSV file and the corresponding rich JSON file and merges them to produce the fine-tuning dataset. 
3. Actually generate such a training set and fine-tune a new model.

### Example fine-tuning dataset entries:

{"`claim`":"Fenugreek may help to reduce body fat especially around the abdominal area.",
"`label`":"false",
 "`reason`":"may is a hedge word"}

{"`claim`":"A whole food plant-based diet can help fight cancer.",
 "`label`":"false",
 "`reason`":"can is a hedge word"}

{"`claim`":"HPV is associated with almost 100% of cervical cancers.",
"`label`":"false",
 "`reason`":"claim is accurate"}

{"`claim`":"Cinnamon is one of the most effective cancer fighting food mainly because it has high antioxidant content and also because of its antibacterial property.",
"`label`":"true",
 "`reason`":"high harm"}

{"`claim`":"In many Scandinavian countries, even if you get aggressive cancer, they still don't treat you, they'll just watch you.",
 "`label`":"true",
 "`reason`":"high harm"}

**Comment by dcorney** [comment]
[First version done](https://github.com/FullFact/health-misinfo-shared/commit/35f74081d9950ece8396b4fd0e349e0963780132)
But still need to refine / decide on labels to use.
